packages/server/src/helpers.ts

```diff
@@ -5,26 +5,13 @@ export function matchCurrencies(
   currencies: Currency[],
   patterns: string[],
 ): Currency[] {
-  const matchedCurrencies: Currency[] = [];
-  const patternCount = patterns.length;
-  const currencyCount = currencies.length;
+  const matchers = patterns
+    .filter((pattern) => pattern)
+    .map((pattern) => picomatch(pattern));
```

Comment:
We can probably do both filtering and mapping in one loop
```suggestion
  const matchers = patterns.reduce((filtered, pattern) => {
    if (pattern) {
      filtered.push(picomatch(pattern));
    }
    return filtered;
  }, []);
```